src/utils/shop-validator.ts

```diff
@@ -4,6 +4,7 @@
  * @param shop Shop url: {shop}.myshopify.com
  */
 export default function validateShop(shop: string): boolean {
-  const shopUrlRegex = /^[a-zA-Z0-9][a-zA-Z0-9-]*\.myshopify\.(com|io)[/]*$/;
+  const shopUrlRegex =
+    /(^[a-zA-Z0-9][a-zA-Z0-9-]*\.myshopify\.(com|io)[/]*$|^[a-zA-Z0-9][a-zA-Z0-9-]*\.shopify\.[a-zA-Z0-9][a-zA-Z0-9-]*\.[a-zA-Z0-9][a-zA-Z0-9-]*\.(us|asia|eu)\.spin.dev[/]*$)/;
```

Comment:
The regex works for me but it's a bit hard to read in this format. Do you think it would be easier to read if we put each of the individual components into strings and then just `|` them in the final regex?

Reply:
For sure I can split them up into different variables 👍🏼